### PR TITLE
Update open source community files

### PR DESCRIPTION
**Pull request by Jonasvdbo**
Updated the readme.md and added the following open source community files:

CONTRIBUTING.md
CODE_OF_CONDUCT.md
PROJECT_GOVERNANCE.md
RELEASE.md
SUPPORT.md

### CODE REVIEW

**Comment by Jonasvdbo** [comment]
@bramstoeller I am just following up to check what you think of my PR

**Comment by bramstoeller** [comment]
> @bramstoeller I am just following up to check what you think of my PR

Hi @Jonasvdbo , sorry, I completely missed your PR and message 🙈 

I've asked @nitbharambe to review it, he's probably better informed on these matters.

**Comment by Jonasvdbo** [comment]
@nitbharambe I am just following up to check what you think of my PR?

**Comment by nitbharambe** [comment]
Hi Jonas,
We dont follow any release strategy in `RELEASE.md` as of now. Also the `PROJECT_GOVERNANCE.md` seems to be valid only for power-grid-model repository.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+This project follows the following [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## REUSE Compliance
+
+All the files in the repository need to be [REUSE compliant](https://reuse.software/). 
+We use the pipeline to automatically check this.
+If there are files which are not complying, the pipeline will fail the pull request will be blocked.
+
+
+## Git branching
```
Comment (nitbharambe) [inline CONTRIBUTING.md:38]: This too would related be related to the content `RELEASE.md` right?

Thanks for contributing by the way!

**Comment by Jonasvdbo** [comment]
@nitbharambe I removed the Release.md and updated the PROJECT_GOVERNANCE.md.

File: CONTRIBUTING.md (hunk 1 above)
Comment (nitbharambe) [reply CONTRIBUTING.md:38]: Okay, I think it makes sense keep this.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+## Filing bugs and change requests
+
+You can file bugs against and change request for the project via GitHub issues. Consult [GitHub Help](https://docs.github.com/en/free-pro-team@latest/github/managing-your-work-on-github/creating-an-issue) for more
+information on using GitHub issues.
+
+## Community Guidelines
+
+This project follows the following [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## REUSE Compliance
```
Comment (nitbharambe) [inline CONTRIBUTING.md:31]: But lets remove the `REUSE Compliance`. There is no pipeline as of now to check it.
Thanks again!

Comment (Jonasvdbo) [reply CONTRIBUTING.md]: Sure

**Comment by Jonasvdbo** [comment]
@nitbharambe Removed the REUSE compliance in CONTRIBUTING.md

**Review by nitbharambe — APPROVED** [review]

**Pull request merged**